resources/definitions/deltacomb.def.json

```diff
@@ -25,41 +25,32 @@
-        "machine_max_jerk_e":                    { "default_value": 1.0 },
-        "retraction_hop_enabled":                { "default_value": false },
-        "retraction_amount" :                    { "default_value": 4.5 },
-        "retraction_speed" :                     { "default_value": 40 },
+        "retraction_hop_enabled":                { "default_value": true },
+        "retraction_amount" :                    { "default_value": 3.5 },
+        "retraction_speed" :                     { "default_value": 50 },
         "material_diameter":                     { "default_value": 1.75 },
         "material_final_print_temperature":      { "value": "material_print_temperature - 5" },
         "material_initial_print_temperature":    { "value": "material_print_temperature" },
         "material_print_temperature_layer_0":    { "value": "material_print_temperature + 5" },
         "travel_avoid_distance":                 { "default_value": 1, "value": 1 },
-        "speed_print" :                          { "default_value": 70 },
-        "speed_travel":                          { "default_value": 80, "value": 80 },
+        "speed_print" :                          { "default_value": 60 },
+        "speed_travel":                          { "value": 150.0 },
         "speed_infill":                          { "value": "round(speed_print * 1.05, 0)" },
         "speed_topbottom":                       { "value": "round(speed_print * 0.95, 0)" },
         "speed_wall":                            { "value": "speed_print" },
-        "speed_wall_0":                          { "value": "round(speed_print * 0.9, 0)" },
+        "speed_wall_0":                          { "value": 20 },
         "speed_wall_x":                          { "value": "speed_wall" },
         "speed_layer_0":                         { "value": "min(round(speed_print * 0.75, 0), 45.0)" },
         "speed_travel_layer_0":                  { "value": "round(speed_travel * 0.7, 0)" },
         "skirt_brim_speed":                      { "value": "speed_layer_0" },
         "skirt_line_count":                      { "default_value": 3 },
         "skirt_brim_minimal_length":             { "default_value": 150 },
-        "infill_sparse_density":                 { "default_value": 24 },
-        "top_bottom_thickness":                  { "default_value": 0.6 },
-        "support_z_distance":                    { "default_value": 0.2, "value": "min(2 * layer_height, machine_nozzle_size * 0.75)" },
+        "infill_sparse_density":                 { "default_value": 90 },
+        "gradual_infill_steps":                  { "default_value": 2 },
         "infill_before_walls" :                  { "default_value": false },
+        "top_bottom_thickness":                  { "default_value": 0.6 },
+        "support_z_distance":                    { "value": "layer_height * 2" },
+				"support_bottom_distance":               { "value": "layer_height" },
```

Comment:
Aww, it was all so neatly indented before...

Reply:
Gosh! I missed it! Can I fix it?